PennMobile/New Group/PAC Code/PacCodeViewController.swift

```diff
@@ -103,38 +143,32 @@ extension PacCodeViewController : KeychainAccessible, LocallyAuthenticatable {
     func handleAuthenticationSuccess() {
         if let pacCode = getPacCode() {
             self.pacCode = pacCode
+            updatePACCode()
         } else {
             if Account.isLoggedIn {
-                self.showActivity()
-                
                 // Handle the case in which the user is logged in but hasn't yet fetched their PAC Codes
-                // Acts as though the user pressed the refresh button
-                PacCodeNetworkManager.instance.getPacCode { result in self.handleNetworkPacCodeRefreshCompletion(result) }
+                handleNetworkPacCodeRefetch()
```

Comment:
The code that dealt with showing the activity indicator, refetching of the PAC code using the network manager, handling the response, and finally hiding the indicator has been refactored away into a new function called `handleNetworkPacCodeRefetch()`  in line 180 -185.